contrib/inventory_builder/inventory.py

```diff
@@ -179,6 +180,23 @@ def build_hostnames(self, changed_hosts):
 
         return all_hosts
 
+    def range2ips(self, hosts):
+        reworked_hosts = []
+
+        def ips(start_address, end_address):
+            from ipaddress import ip_address
+            start = int(ip_address(start_address).packed.hex(), 16)
+            end = int(ip_address(end_address).packed.hex(), 16)
+            return [ip_address(ip).exploded for ip in range(start, end+1)]
+
+        for host in hosts:
+            if '-' in host:
```

Comment:
Can we wrap this in a try/except block? I want to print an error and exit if an ip address range is specified wrong